cl/favorites/utils.py

```diff
@@ -153,8 +167,14 @@ async def get_top_prayers() -> QuerySet[RECAPDocument]:
                 "prayers", filter=Q(prayers__status=Prayer.WAITING)
             ),
             view_count=F("docket_entry__docket__view_count"),
+            doc_unavailable=Exists(doc_unavailable),
```

Comment:
Can we use the `is_sealed` field instead? Every sealed document with prayers should have a record in the `PrayerAvailability` table, right?

Reply:
Good catch! Thanks